client/src/pages/admin/Admin/Admin.jsx

```diff
@@ -1,70 +1,19 @@
 import React from "react";
-import { Routes, Route } from "react-router-dom";
-import { useDispatch, useSelector } from "react-redux";
-import { useEffect, useState } from "react";
-import { setContentVersions } from "../../../redux/actions/contentVersions";
-import HomePage from "../../public/HomePage/HomePage";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { styles } from "../../../variable-css";
+import { makeStyles } from '@mui/styles'
+// import AdminHomePage from "./AdminHomePage";
+// import AdminProfilePage from "./AdminProfilePage";
+// import Authenticate from "../../components/Auth/Authenticate";
 
-import AdminProfilePage from "../AdminProfilePage";
-import AdminHomePage from "../AdminHomePage/AdminHomePage";
-import SectionView from "../../public/SectionPage/SectionView";
+const useStyles = makeStyles(styles)
 
-function Admin(){
-    const [currentSections, setCurrentSections] = useState([]);
-    const [currentUserProfile, setCurrentProfile] = useState({});
+function Admin() {
+    const classes = useStyles()
 
-
-    const currentUser = "Cynaptics"
-
-    const dispatch = useDispatch();
-    useEffect(()=>{
-        dispatch(setContentVersions(currentUser))
-    },[dispatch])
-
-    let contentVersions = useSelector((state)=> state.contentVersions)
-
-    useEffect(() => {
-        try {
-            let userName = currentUser
-
-            let currentVersion = contentVersions[(contentVersions).length - 1]
-
-            let currentName = currentVersion.userDetails.name
-            let currentLogoSrc = currentVersion.userDetails.logo
-            let currentSocialMedia = currentVersion.userDetails.socialMedia
-
-            setCurrentSections(currentVersion.Sections)
-
-            let currentEmail = currentVersion.contactDetails.email
-            let currentPhoneNumber = currentVersion.contactDetails.phoneNumber
-
-            let currentHomePagePoster = currentVersion.homePagePoster
-            let currentThemeDetails = currentVersion.themeDetails
-
-            setCurrentProfile({
-                "userName":userName,"name": currentName, "email": currentEmail, "logo": currentLogoSrc, "socialMedia": currentSocialMedia, "phoneNumber": currentPhoneNumber,
-                "src": currentHomePagePoster.src, "caption": currentHomePagePoster.caption, "theme": currentThemeDetails
-            })
-
-        } catch (error) {
-            console.log(error)
-        }
-    }, [contentVersions])
-
-    return(
+    return (
         <>
-        {
```

Comment:
Why deleted this part?

Reply:
Sorry bhaiya, acutally it may have changed while I was working, and when I pushed to origin, it caused some conflicts, which resulted in this.